src/image/utils/custom_yaml.py

```diff
@@ -0,0 +1,14 @@
+import yaml
+
+
+def string_constructor(loader, node):
+    """change node type to string"""
+    return loader.construct_scalar(node)
+
+
+yaml.add_constructor(
+    "tag:yaml.org,2002:int", string_constructor, Loader=yaml.SafeLoader
```

Comment:
Because next time someone writes a new yaml.load in the code, they may not be aware (or they may forget) that they should use different loader. Hence I opted for the approach, so that all current (and futuristic) yaml loads will use the custom string constrcutor without any further modification on yaml method calls (aka yaml.safe_load).

Reply:
that is something a unit test should solve (which is yet something we're missing).

I'd rather keep it simple. This approach is introducing a lot of relative imports and unused imports.

Reply:
done